src/lib/stores/walletStore.js

```diff
@@ -16,11 +16,48 @@ import { TransactionBuilder } from 'stellar-sdk'
 
 function createWalletStore() {
     /** @type {import('svelte/store').Writable<WalletStore>} */
-    const { subscribe, set } = persisted('bpa:walletStore', { keyId: '', publicKey: '' })
-
+    const { subscribe, set, } = persisted('bpa:walletStore', { keyId: '', publicKey: '' })
     return {
         subscribe,
 
+         /**
+         * Checks if a wallet is already registered
+         * @param {string} publicKey Public Stellar address
+         * @returns {Promise<boolean>} True if the wallet is registered, false otherwise
+         */
```

Comment:
looks like this is leftover from a previous commit. Let's remove the unnecessary jsdoc comment